cves/CVE-2015-8213.yml

```diff
@@ -105,10 +105,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: No units tests were used in the fix for this cve.
+  fix: false
```

Comment:
It looks like a test was added in the fix commits. Also, you should put notes for the fix commits.